app/controllers/organisations_controller.rb

```diff
@@ -107,12 +107,14 @@ def lettings_logs
   def download_csv
     organisation_logs = LettingsLog.all.where(owning_organisation_id: @organisation.id)
     unpaginated_filtered_logs = filtered_logs(organisation_logs, search_term, @session_filters)
+    codes_only = params.require(:codes_only) == "true"
 
-    render "logs/download_csv", locals: { search_term:, count: unpaginated_filtered_logs.size, post_path: logs_email_csv_organisation_path }
+    render "logs/download_csv", locals: { search_term:, count: unpaginated_filtered_logs.size, post_path: logs_email_csv_organisation_path, codes_only: }
   end
```

Comment:
Do we need to authorise for this too?

Reply:
how did you write a comment on a line that hadn't been changed? I can't seem to be able to.

line 150 you have an existing authenticate_scope that covers this only being available for support users. This is fine since if you're not support you are directed through the lettings log controller anyway

Reply:
You can select several lines, but maybe because some of the lines I've selected have been changed that's why it let me comment on them

Nice